pip_audit/_fix.py

```diff
@@ -0,0 +1,92 @@
+"""
+Resolving fix versions.
+"""
+
+from dataclasses import dataclass
+from typing import Dict, Iterator, List, cast
+
+from packaging.version import Version
+
+from pip_audit._service import (
+    Dependency,
+    ResolvedDependency,
+    VulnerabilityResult,
+    VulnerabilityService,
+)
+
+
+@dataclass(frozen=True)
```

Comment:
I've followed a similar pattern to how we handle dependencies if you recall that @woodruffw.

Reply:
Yep, this LGTM!